.github/workflows/push-image.yml

```diff
@@ -49,14 +45,14 @@ jobs:
         DOCKERHUB_ORG="${GITHUB_REPOSITORY_OWNER/-/}" # translates 'paketo-buildpacks' to 'paketobuildpacks'
         # Strip off the Github org prefix from repo name
         # paketo-buildpacks/builder-with-some-name --> builder-with-some-name
```

Comment:
```suggestion
```

Reply:
I think the comments are no longer necessary as the registry_repo variable is hardcoded